### PR TITLE
HwF - Ithc setup

### PR DESCRIPTION
**Pull request by zaparka**

### CODE REVIEW

File: apps/help-with-fees/serviceaccount/ithc.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ${NAMESPACE}
+  namespace: ${NAMESPACE}
+  annotations:
+    azure.workload.identity/client-id: ""
```
Comment (thomast1906) [inline apps/help-with-fees/serviceaccount/ithc.yaml:7]: you probably need to have a value here if you are wanting to use that?

Comment (zaparka) [reply apps/help-with-fees/serviceaccount/ithc.yaml:7]: @thomast1906 I'm in the process of finding the value for it.

Comment (thomast1906) [reply apps/help-with-fees/serviceaccount/ithc.yaml:7]: Sure - we will hold off merging until value added

**Review by garytaylor — APPROVED** [review]

File: apps/help-with-fees/serviceaccount/ithc.yaml (hunk 1 above)
Comment (zaparka) [reply apps/help-with-fees/serviceaccount/ithc.yaml]: @thomast1906 ready to merge

**Review by timja — APPROVED** [review]

**Pull request merged**